### ISSUE TITLE
CSV report

### ISSUE THREAD

**Issue by msaroufim** [sgugger/torchdynamo-tests#5]
I'm running a bunch of long running tests in the background so might be useful for us to have something like this https://gist.github.com/msaroufim/3278d9af9efd77007bfb8aeef8c05f7d

If this is interesting lemme know and I can send the PR out

```python
    out_dict = {
            "backend" : args.dynamo_backend,
            "num_epochs" : str(args.num_epochs),
            "seed" : str(args.seed),
            "train_acc" : str(eval_train_metric["accuracy"]),
            "train_f1" : str(eval_train_metric["f1"]),
            "avg_train_time" : str(avg_train_iteration_time * 1000),
            "test_acc" : str(eval_test_metric["accuracy"]),
            "test_f1" : str(eval_test_metric["f1"]),
            "avg_test_time" : str(avg_test_iteration_time * 1000),


    }

    with open("results.csv", "a") as fd:
        fd.write("\n")
        fd.write(",".join(out_dict.values()))
```

That way I can run the below at night

```bash
for i in {0..50}
do
    echo $i
    TOKENIZERS_PARALLELISM=false accelerate launch --config_file configs/base_fp32.yaml scripts/text_classification.py --task_name mrpc --dynamo_backend inductor --seed $i --num_epochs 3
    TOKENIZERS_PARALLELISM=false accelerate launch --config_file configs/base_fp32.yaml scripts/text_classification.py --task_name mrpc --seed $i --num_epochs 3
done
```

My hypothesis is that inductor is more sensitive to the random seed than eager with higher variance so hoping this will confirm that

**Comment by sgugger** [comment]
Anything that helps debug the issue! This repo is meant for that :-)

**Comment by msaroufim** [comment]
Code was improved and merged
